ait/core/server/handler.py

```diff
@@ -125,18 +123,24 @@ def handle(self, input_data):
         Returns:
             tuple of packet UID and packet data field
         """
+
         packet = bytearray(input_data)
+        if (len(packet) < 13):
+            ait.core.log.info('CCSDSPacketHandler: Insufficient packet length.')
```

Comment:
This is good but I'd like to see a more descriptive message. "Insufficient packet length" isn't wrong but something such as "Received packet is less than minimum of 12 bytes. Cannot process CCSDS header / packet properly" or something equally verbose points the user in the correct direction and keeps this error distinct from the other "Insufficient packet length" error(s)